specs/sb/api.md

```diff
@@ -573,9 +573,9 @@ existing system.
 </thead>
 <tbody>
 <tr>
-  <td>organization_guid*</td>
+  <td>context_guid*</td>
   <td>string</td>
-  <td>The Cloud Controller GUID of the organization under which the service is to be provisioned. Although most brokers will not use this field, it could be helpful in determining data placement or applying custom business rules.</td>
+  <td>The Cloud Controller GUID of the context under which the service is to be provisioned. Although most brokers will not use this field, it could be helpful in determining data placement or applying custom business rules.</td>
```

Comment:
I think it might be helpful to provide an example here, both of how this would be used within CF and another PaaS/orchestrator.